tests/virtualization/xen/register_guests.pm

```diff
@@ -28,7 +28,10 @@ sub run {
 
     foreach my $guest (keys %xen::guests) {
         record_info "$guest", "Registrating $guest against SMT";
-        assert_script_run("ssh root\@$guest 'zypper ar --refresh http://download.suse.de/ibs/SUSE:/CA/" . $xen::guests{$guest}->{distro} . "/SUSE:CA.repo'", 90);
+        assert_script_run("ssh root\@$guest SUSEConnect -r " . get_var('SCC_REGCODE') . " -e " . get_var("SCC_EMAIL"));
```

Comment:
The `SUSEConnect` is not available on old systems:
```suggestion
        # There's no SUSEConnect on SLE 11
        ($sles_running_version, $sles_running_sp) = get_sles_release("ssh root\@$guest");
        if ($sles_running_version >= 12) {
            assert_script_run("ssh root\@$guest SUSEConnect -r " . get_var('SCC_REGCODE') . " -e " . get_var("SCC_EMAIL"));
        }
```
Also add `use version_utils;` before `sub run {` and `my ($sles_running_version, $sles_running_sp);` before `foreach my $guest (keys %xen::guests) {`

Reply:
Done